source/_components/nzbget.markdown

```diff
@@ -16,7 +16,7 @@ To use NZBGet with your installation, add the following to your `configuration.y
 
 ```yaml
 # Example configuration.yaml entry
-sensor:
+nzbget:
   platform: nzbget
```

Comment:
I don't think this line is correct though...

Reply:
I'll fix.